examples/hybrid/driver.jl

```diff
@@ -128,7 +120,6 @@ function additional_cache(Y, parsed_args, params, atmos, dt;)
             Y,
             turbconv_model,
             atmos,
-            namelist,
```

Comment:
Since now we have the `params` for numbers and `config` for flags, would it make sense (at some point) to move the data from `parsed_args` into either of the two and pass them into `turbcov_cache`, etc?